docs/advanced/api.rst

```diff
@@ -27,9 +27,34 @@ To configure firewalld to allow access to the control service REST API, and for
 (On AWS, an external firewall is used instead, which will need to be configured similarity).
 For more details on configuring the firewall, see Fedora's `firewalld documentation <https://fedoraproject.org/wiki/FirewallD>`_.
 
-To start the agents on a node, (where ``${CONTROL_NODE}`` is the address of the control node, and ``${NODE_NAME}`` is the name of the node being configured).:
+To start the agents on a node, a configuration file must exist on the node at ``/etc/flocker/dataset-agent.yml``.
+This file has the format:
 
-.. task:: enable_flocker_agent ${NODE_NAME} ${CONTROL_NODE}
+
+.. code-block:: yaml
+
+   node-name:
+   control-service-endpoint: tcp:control-service-hostname:4524
+   dataset:
+      backend: zfs
+      zfs-pool: flocker
+
+.. Remove this before doing a release (JIRA-XXX)
+
+or if loopback, dataset key is like:
+
+.. code-block:: yaml
+
+   dataset:
+      backend: loopback
```

Comment:
This configuration file needs to be versioned so we can change it easily.

I'm also not sure `zfs` vs `loopback` reflects the right abstraction.  It's probably "peer-to-peer" vs "block-device" with "zfs" and "loopback" as deeper configuration information supplied to each of those backends.

This might not matter, though.  Particularly if the file is versioned such that we can easily make big changes to it.